update_messages_with_surveys/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+CoreDataModules = {editable = true, ref = "v0.7.0", git = "https://www.github.com/AfricasVoices/CoreDataModules"}
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
```

Comment:
No new line at end of file. There should be a switch in your editor you can flick to prevent this from ever happening again.